### ISSUE TITLE
make distcheck warning

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-ldns#95]
```
make distcheck
/usr/bin/perl "-Iinc" "-MExtUtils::Manifest=fullcheck" -e fullcheck
Not in MANIFEST: LDNS.bs
```

**Comment by mattias-p** [comment]
I'm able to reproduce this if I run `make all` twice. However if I only run `make all` once, I don't get this warning.

**Comment by sandoche2k** [comment]
I did run only once `make all`

**Comment by mattias-p** [comment]
I'm not sure what the bs file is but it seems it's only used as an intermediate when generating the LDNS.so file. I'm also not sure why the Makefile shows this strange behavior with regard to the bs file.
However it seems harmless to include the bs file in the MANIFEST.SKIP file and that should remove the warning you're seeing.

**Comment by matsduf** [comment]
Resolved by #101.
